game.py

```diff
@@ -106,6 +128,7 @@ def play_card(self, card):
             self.turn()
         else:
             self.logger.debug("Choosing Color...")
+            self.logger.info("Played special card with hand: {hand}".format(hand=self.current_player.cards))
```

Comment:
Do we need this log?

Reply:
Nope, not really. That's there because I have a friend that said "hey, I didnt bluff!" a couple times, with this I can prove him wrong. I'll remove it for the next pull request.